src/components/Footer/index.tsx

```diff
@@ -21,6 +24,8 @@ interface FooterProps {
   developers: string | Array<string>
   /** The year that the story was published, e.g., 2018. */
   copyrightYear: number
+  /** What type of project this footer is being used in. */
+  footerType: LuxComponentType
```

Comment:
Yeah - looks like context API will really help this use case! I'll create an issue for this so it can be updated once we have our discussion in a future PR?